aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -45,6 +45,16 @@
         }
       }
     },
+    "ChannelUris": {
+      "description": "The Amazon Resource Name (ARN) of the specified SNS Topic.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Arn",
+        "examples": [
+          "arn:aws:sns:us-east-1:977364881114:MyTopic"
+        ]
+      }
+    },
```

Comment:
While reasonable, this does not line up with the [`AddNotificationChannels`](https://docs.aws.amazon.com/codeguru/latest/profiler-api/API_AddNotificationChannels.html) API, nor the [`Channel`](https://docs.aws.amazon.com/codeguru/latest/profiler-api/API_Channel.html) object that we have.

My suggestion is to -- similarly with what we did when turning the `PutPermission` API into `AgentPermissions` above is to name this according to the specific `AnomalyDetection` option we have for the notifications API.

That is, have a `AnomalyDetectionNotificationConfiguration` entry that inside it has an array of items, where each item can can the uri and the optional `id`.

Furthermore, this item should be **optional** so as to not break existing customers already using CloudFormation. Basically any new item added under an existing type after its initial release MUST be optional.